dataclients/kubernetes/testdata/routegroups/loadbalancer-algorithm/default-ch-algorithm.eskip

```diff
@@ -0,0 +1,6 @@
+kube_rg__default__myapp__all__0_0:
+	Host("^(example[.]org[.]?(:[0-9]+)?)$")
+	&& PathSubtree("/")
+	-> <roundRobin, "http://10.2.9.103:7272", "http://10.2.9.104:7272">;
```

Comment:
:boom:

Reply:
fix in https://github.com/zalando/skipper/pull/2614